joomla/joomla-cms
# Adds "Category:" text to contacts, banners and newsfeeds lists

#### Description

This PR adds "Category:" text to contacts, banners and newsfeeds list views just like there is in articles list view.

Before patch:
![image](https://cloud.githubusercontent.com/assets/9630530/12081472/0c315340-b272-11e5-896a-2f919705d61c.png)

After patch:
![image](https://cloud.githubusercontent.com/assets/9630530/12081470/f9e1163a-b271-11e5-986f-13fcb6a18b8c.png)
#### How to test
1. Go to contacts, banners and newsfeeds list views and check that each item has only the category name below the item title.
2. Apply the patch
3. Go to contacts, banners and newsfeeds list views and check that each item has "Category: " plus the category name below the item title, like in the articles list view.
